partcad/src/partcad/part_factory_scad.py

```
@@ -42,7 +61,59 @@ async def instantiate(self, part):
+            )
+            if len(errors) > 0:
+                error_lines = errors.split("\n")
+                for error_line in error_lines:
+                    part.error("%s: %s" % (part.name, error_line))
+
+            try:
+                # pc_logging.info("Response: %s" % response_serialized)
+                response = base64.b64decode(response_serialized)
+                register_ocp_helper()
+                result = pickle.loads(response)
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)